webapp-client/src/component/Navbar.js

```diff
@@ -0,0 +1,33 @@
+import logo from '../lobster-icon.jpg';
+import React, { Component } from 'react';
+
+class Navbar extends Component {
+  render() {
+    return (
+      <nav
+        className="navbar mb-1"
+        role="navigation"
+        aria-label="main navigation"
+      >
+        <div className="navbar-brand">
+          <a className="navbar-item">
+            <img
+              src={logo}
+              alt="Lobster Ice Cream Logo (a cute lobster)"
+              className="is-32x32"
+            />
+          </a>
+          <a className="navbar-item">
+            <h1 className="title is-4">Lobster Ice Cream</h1>
+          </a>
+        </div>
+        <div className="navbar-end">
+          {/* TODO: Put the user's actual username */}
+          <div className="navbar-item">Username</div>
+        </div>
+      </nav>
+    );
+  }
+}
+
+export default Navbar;
```

Comment:
also for consistency hehe you can get rid of this line and do "export default class Navbar..." when you first declare the class on line 4 (same goes for Item.js)

Reply:
done

Reply:
Done